packages/appstore/src/components/options-multipliers-listing/index.tsx

```diff
@@ -27,7 +29,7 @@ const OptionsAndMultipliersListing = observer(() => {
     const cr_demo = content_flag === ContentFlag.CR_DEMO;
 
     const OptionsTitle = () => {
-        if (is_mobile) return null;
+        if (!isDesktop) return null;
```

Comment:
the hook also provide isMobile. why didn't u just use that?

Reply:
@shayan-deriv it's because Tablet view uses the same components as for mobile view. So in this case `!isDesktop` means `isMobile` OR `isTablet`

Reply:
oh I see. so later when the design is changed for those screens then u will use them right?

Reply:
@shayan-deriv honestly, I don't know what will be in the future :) In my opinion, we just need to change `is_mobile` and `is_desktop` values in the `ui-store`, also change `isMobile(), isTablet() and isDesktop()` values for these utility function in `shared` package. But for now we need to add more breakpoints for mixins and use useDevice hoook from deriv-com/ui. Also, Tablet design completed just for one page :) So I don't know what it will be in the future))))

Reply:
I wish we had more details before starting this task

Reply:
@shayan-deriv me too :)